atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasBackupService.java

```diff
@@ -167,7 +171,8 @@ private static LockRefresher<InProgressBackupToken> getLockRefresher(
      * Must be called after any failure; after calling this, completeBackup will fail for any namespace which
      * has not yet reached that stage.
      */
-    public void cleanupBackup() {
+    @Override
+    public void close() {
```

Comment:
[Sanity check]: As written, this method is idempotent. Is that the case for all calls inside this method?